docs/errors/E0326.md

```diff
@@ -0,0 +1,20 @@
+# E0326: 'async export' is not allowed; write 'export async' instead
+
+Class methods can have modifiers such as `export`, `static`, `async`, and `*`. It is a
+syntax error to write `async export` in a function declaration:
+
+```javascript
+async export function f() {
+  return 0;
+}
+```
+
+To fix this error, replace `async export` with `export async`:
+
+```javascript
+export async function f() {
+  return 0;
+}
+```
+
+Introduced in quick-lint-js version 3.0.0.
```

Comment:
Must fix: Remove this line. I don't know what the next version number will be yet.